salt/states/consul.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+'''
+Consul Managemnt
+===========================
```

Comment:
This header (`====`) needs to match the number of characters in the `Consul Management` title, otherwise the doc build will output a warning.

There's also a typo in the heading. `Management`.